docs/awesome-pets/papers/applications/ppml/fl/survey.md

```diff
@@ -9,6 +9,8 @@
 
 - [Vertical Federated Learning](https://arxiv.org/pdf/2211.12814.pdf)
 
+- [Vertical Federated Learning: A Structured Literature Review](https://arxiv.org/pdf/2212.00622.pdf)
+
```

Comment:
Please recommend articles that are acccepted by CCF B or above.

Reply:
``[Vertical Federated Learning](https://arxiv.org/pdf/2211.12814.pdf)`` is also a paper in arxiv and I think the structured literature review maybe can be accept as it was introduction above.